src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [filteredMovies, setFilteredMovies] = useState(moviesFromServer);
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    const visibleMovies = filteredMovies
```

Comment:
```suggestion
    const visibleMovies = moviesFromServer
```
цемало б пофіксити проблему з відео